### ISSUE TITLE
Missing script on camera of MiniRig

### ISSUE THREAD

**Issue by thecrazycraft** [nukadelic/unity-application-spacewarp#8]
The MiniRig's camera has a missing script. Using Unity 2021.3.22f1, fresh install, nothing changed

**Comment by nukadelic** [comment]
yeah i keep forgetting to assign it , just set reference to main camera . easy fix
